lib/mutant/meta/example.rb

```diff
@@ -1,7 +1,7 @@
 module Mutant
   module Meta
     class Example
-      include Adamantium, Concord::Public.new(:file, :node, :mutations)
+      include Adamantium, Anima.new(:file, :node, :node_type, :expected)
```

Comment:
> Yeah, I was just saying if Anima is doing it in-code, maybe it would be worth considering pushing it down to have ruby itself handle it. I'm not sure if there's a way to do this without "string class_eval" though -- it's kind of sad there's no nice interface to assemble some kinds of ruby methods without resorting to string class_eval.

The `kwargs` are just syntax for ruby as in anima. There is AFAIK no ruby interface for checking hash contents, or in-lining AST nodes as behavior.